gammapy/maps/core.py

```diff
@@ -167,7 +167,7 @@ def create(**kwargs):
             raise ValueError(f"Unrecognized map type: {map_type!r}")
 
     @staticmethod
-    def read(filename, hdu=None, hdu_bands=None, map_type="auto", format=None):
+    def read(filename, hdu=None, hdu_bands=None, map_type="auto", format=None, colname=None):
```

Comment:
Please extend the docstring to list the `colname` argument and provide a one line description of what it is...